user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/MaintenanceSettings_xml.md

```diff
@@ -214,6 +214,23 @@ The following types of alarm events will not be combined in a consolidated alarm
 > - [Alarm linking](xref:Alarm_linking)
 > - [Declutter your alarm tree](https://community.dataminer.services/declutter-your-alarm-tree/ "Alarm squashing")
 
+### AlarmSettings.PersistParameterLatchState
+
+From DataMiner 10.4.9/10.5.0 onwards<!--RN 39495-->, parameter latch states are no longer persistent by default. This means they will reset after every DataMiner restart.
+
+To enable persistent parameter latch states, add the following tags and values in *MaintenanceSettings.xml*:
+
+```xml
+<AlarmSettings>
+   <PersistParameterLatchState>true</PersistParameterLatchState>
+</AlarmSettings>
+```
+
+> [!NOTE]
+>
+> - This setting will only be applied after a DMA restart.
```

Comment:
As changes to MaintenanceSettings.xml always require a DMA restart (as mentioned at the top of this document), it might be better not to mention this again here. Otherwise people might get the impression that some changes need a restart, and some don't.